model/tensorflow_hub/README.md

```diff
@@ -0,0 +1,59 @@
+# DFFML model_name Models
```

Comment:
Let's make sure we update the README, instead of the `Usage` section you can link to the plugins documentation page. Same as `tensorflow` does: https://github.com/intel/dffml/tree/master/model/tensorflow#documentation